src/daemon/command_parser_executor.cpp

```diff
@@ -602,7 +602,43 @@ bool t_command_parser_executor::ban(const std::vector<std::string>& args)
       return false;
     }
   }
-  return m_executor.ban(ip, seconds);
+  if (boost::starts_with(args[0], "@"))
+  {
+    const std::string ban_list = args[0].substr(1);
+      const boost::filesystem::path ban_list_path(ban_list);
```

Comment:
I hate myself for commenting on this, but the spacing switches from 2 to 4 here.